server/routerlicious/packages/lambdas-driver/src/kafka-service/partition.ts

```diff
@@ -96,7 +96,9 @@ export class Partition extends EventEmitter {
 			return;
 		}
 
-		void this.q.push(rawMessage);
+		this.q.push(rawMessage).catch((error) => {
+			Lumberjack.error("Error pushing raw message to queue", undefined, error);
```

Comment:
nit: to tell this log apart from the one above in documentPartition.ts, maybe we can slightly rephrase the error message to:

`"Error pushing raw message to queue in partition"`